src/main/java/com/sonyericsson/jenkins/plugins/bfa/GerritMessageProviderExtension.java

```diff
@@ -98,5 +98,12 @@ private void addFailureCausesFromData(StringBuilder message, FailureCauseDisplay
             .append(displayData.getLinks().getBuildUrl())
             .append(" )");
         }
+
+        if (displayData.getDownstreamFailureCauses().isEmpty() && displayData.getFoundFailureCauses().isEmpty()) {
+            message.append("Unknown (")
```

Comment:
I decided to use "configured noCausesMessage". At least in this case users have option to change it.